src/lib/NonlinearDutchDecayLib.sol

```diff
@@ -8,19 +8,20 @@ import {MathExt} from "./MathExt.sol";
 import {Uint16ArrayLibrary, Uint16Array, fromUnderlying} from "../types/Uint16Array.sol";
 import {DutchDecayLib} from "./DutchDecayLib.sol";
 
-/// @notice thrown when the decay curve is invalid
-error InvalidDecayCurve();
-
 /// @notice helpers for handling non-linear dutch order objects
 library NonlinearDutchDecayLib {
     using FixedPointMathLib for uint256;
     using MathExt for uint256;
     using Uint16ArrayLibrary for Uint16Array;
 
+    /// @notice thrown when the decay curve is invalid
+    error InvalidDecayCurve();
```

Comment:
is this used?

Reply:
Yes, line 34 of this file (if relativeAmounts length is outside of expected range).